### PR TITLE
🌐 [i18n-KO] Translated `main_classes/quantization.md` to Korean

### PR DESCRIPTION
**Pull request by fabxoe**
# What does this PR do?

Translated the `main_classes/quantization.md` file of the documentation to Korean.
Thank you in advance for your review. 

Part of https://github.com/huggingface/transformers/issues/20179

## Before reviewing
- [x] Check for missing / redundant translations (번역 누락/중복 검사)
- [x] Grammar Check (맞춤법 검사)
- [x] Review or Add new terms to glossary (용어 확인 및 추가)
- [x] Check Inline TOC (e.g. `[[lowercased-header]]`)
- [ ] Check live-preview for gotchas (live-preview로 정상작동 확인)

## Who can review? (Initial)

@chhaewxn, @ahnjj, @jun048098, @fabxoe, @nuatmochoi, @heuristicwave

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review? (Final)

@stevhliu May you please review this PR?

### CODE REVIEW

File: docs/source/ko/main_classes/quantization.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+specific language governing permissions and limitations under the License.
+
+⚠️ Note that this file is in Markdown but contain specific syntax for our doc-builder (similar to MDX) that may not be
+rendered properly in your Markdown viewer.
+
+-->
+
+# 양자화[[quantization]]
+
+Quantization techniques reduce memory and computational costs by representing weights and activations with lower-precision data types like 8-bit integers (int8). This enables loading larger models you normally wouldn't be able to fit into memory, and speeding up inference. Transformers supports the AWQ and GPTQ quantization algorithms and it supports 8-bit and 4-bit quantization with bitsandbytes.
```
Comment (ahnjj) [inline docs/source/ko/main_classes/quantization.md:19]:
```suggestion
```
아래 번역 해두신것 같아 원문부분 삭제했습니다~!

File: docs/source/ko/main_classes/quantization.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+Quantization techniques that aren't supported in Transformers can be added with the [`HfQuantizer`] class.
```
Comment (ahnjj) [inline docs/source/ko/main_classes/quantization.md:21]:
```suggestion
```
아래 번역 해두신것 같아 원문부분 삭제했습니다~!

File: docs/source/ko/main_classes/quantization.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+양자화 기법은 가중치와 활성화를 8비트 정수(int8)와 같은 더 낮은 정밀도의 데이터 타입으로 표현함으로써 메모리와 계산 비용을 줄입니다. 이를 통해 일반적으로는 메모리에 올릴 수 없는 더 큰 모델을 로드할 수 있고, 추론 속도를 높일 수 있습니다. Transformers는 AWQ와 GPTQ 양자화 알고리즘을 지원하며, bitsandbytes를 통해 8비트와 4비트 양자화를 지원합니다.
+Transformers에서 지원되지 않는 양자화 기법들은 [`HfQuantizer`] 클래스를 통해 추가될 수 있습니다.
+
+<Tip>
+
+이 [양자화](../quantization) 가이드를 통해서 모델을 양자화하는 방법을 배울 수 있습니다.
```
Comment (ahnjj) [inline docs/source/ko/main_classes/quantization.md:28]:
```suggestion
 모델을 양자화하는 방법은 이  [양자화](../quantization) 가이드를 통해 배울 수 있습니다.
```
좀더 가독성 있어보이게 바꾸었습니다!

**Review by ahnjj — CHANGES_REQUESTED** [review]

**Review by stevhliu — APPROVED** [review]
Nice, thank you!

**Pull request merged**